src/components/groups/group.service.ts

```diff
@@ -55,6 +56,25 @@ export const createGroup = async (req: Request, res: Response, next: NextFunctio
   next()
 }
 
+export const updateGroup = async (req: Request, res: Response, next: NextFunction) => {
+  await Group.query()
+    .patchAndFetchById(req.params.id, {
```

Comment:
I think we can use `.pach(...).findById()` since we don't care about the updated entity.

Reply:
You're completely right, I misunderstood the documentation!